### DISCUSSION TITLE
如何同时写入多个队列的值？

### DISCUSSION THREAD

**Discussion by icejd** [2881099/FreeRedis#69] [Q&A]
public long RPush(string key, params object[] elements)
这个方法
程序中产生了数组
int[] arrid=new int[]{1,2,3,4,5}
我这样调用
 QueueHelper.RPush("test", arrid);
队列里写入的值是
System.Int32[]
用下面的法子则正常，问题是我想不出如何来转换这个值
 QueueHelper.RPush("test", “1”,"2","3","4","5");
求大神指点，我要如何把这个数组传进去，实现批量写入的目的

**Comment by icejd** [comment]
明白了，只支持
object[]

**Comment by 2881099** [comment]
c# 语法无法隐式转换
